pkg/cmd/extension/run/cmd.go

```diff
@@ -209,7 +210,7 @@ Run Envoy extension in the example setup.`,
 			// run example
 			opts.Run.Workspace = workspace
 			opts.Run.Example.Example = example
-			err = examples.RunExample(&runtime.RunContext{
+			err = examples.RunExample(global.HomeDir, global.DebugDir, &runtime.RunContext{
```

Comment:
it is strange, but another file initializes global.HomeDir, so I repeated the pattern here.